From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 4 Mar 2016 00:59:54 +0100
Subject: [PATCH] Add fuel-agent dependencies to Ubuntu mirror.

Add qemu-user-static and binfmt-support to the list of packages
the local mirror of amd64 Ubuntu should contain.

NOTE: This is needed because fuel-agent from upstream mirror does not
list them as dependencies (not patched for cross-debootstrap).

FIXME: Will building/patching fuel-agent using OPNFV mechanism solve
missing deps from local mirror too?

FIXME: Remove udev package from this patch when Ubuntu Trusty on arm64
fixes persistent device names in the udev it ships out of the box.
---
 contrib/fuel_mirror/data/ubuntu.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/fuel_mirror/data/ubuntu.yaml b/contrib/fuel_mirror/data/ubuntu.yaml
index 7bc0c4b..4509a9a 100644
--- a/contrib/fuel_mirror/data/ubuntu.yaml
+++ b/contrib/fuel_mirror/data/ubuntu.yaml
@@ -77,6 +77,7 @@ packages: &packages
      - "acct"
      - "bash-completion"
      - "bc"
+     - "binfmt-support"
      - "build-essential"
      - "cloud-init"
      - "conntrackd"
@@ -113,6 +114,7 @@ packages: &packages
      - "puppet"
      - "python-lesscpy"
      - "python-pip"
+     - "qemu-user-static"
      - "rsyslog-gnutls"
      - "rsyslog-relp"
      - "screen"
@@ -124,6 +126,7 @@ packages: &packages
      - "tmux"
      - "traceroute"
      - "ubuntu-standard"
+     - "udev"
      - "vim"
      - "virt-what"
      - "xinetd"
